README.md

```diff
@@ -164,6 +175,83 @@ ember exam --split=3 --partition=3
 
 **Note 3**: _You must be using Testem version `1.5.0` or greater for this feature to work properly._
 
+### Test Load Balancing
+
+```bash
+$ ember exam --parallel=<num> --load-balance
+```
+
+`load-balance` allows you to load balance test files against. It will order the test files by test type and load balance the ordered test files between the browsers dynamically rather than statically. `parallel` must be used along with `load-balance` to specify a number of browser(s) and `load-balance` can not be used without `parallel` being specified a number of browser(s).
+
+This option was added to version 1.1 to address execution performance when running against a large test suite.
+
+A way of communication between server and browser to send and receive a test module to load is promise based. Its timeout value is set to be 2 seconds and the timeout can be customized by user by adding `asyncTimeout=[timeout]` in passing url or adding to a testem config.
+
+For example, if you specify load-balance and parallel equals 3, then three browser instances will be created and the output will look something like::
+
+```bash
+ok 1 Chrome 66.0 - Browser Id 1 - some test
+ok 2 Chrome 66.0 - Browser Id 2 - some another test
+ok 3 Chrome 66.0 - Browser Id 3 - some the other test
+```
+
+You can also specify the `split` and `partition` options with `load-balance` to load a portion of test modules on multiple CI containers.
+
+```bash
+$ ember exam --split=<num> --partition=<num> --parallel=<num> --load-balance
+```
+
+This command will split test files and load-balance tests from the specified partition across the browsers. For example --split=2 -partition=1 --parallel=3 --load-balance, the complete list of test files are split into two halves. With the first half of the list load balanced against three browsers. The output will look something like below:
+
+```bash
+ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
+ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - another test
+ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
+```
+
+Due to the dynamic nature of the `load-balance` option, test file execution order could vary between runs. To reproduce a past test execution the execution must be recorded and passing `--write-execution-file`, in short `--wef` allows generating a JSON file which enables to rerun the past test execution. The option is only allowed when `load-balance` is passed.
```

Comment:
This paragraph here seems like a repeat with the first paragraph in `test failure reproduction`, do you feel we should combine them?